playbooks/ood.yml

```diff
@@ -361,6 +361,14 @@
       regexp: 'limit: 1,$'
       replace: 'limit: 1, timeout: 0,'
 
+  - name: Enable file extension restriction in Uppy
```

Comment:
It would be better to execute this only if ondemand.file_upload_allowed_extensions is defined, by using a when clause on the task itself, so that it will be skipped and that would be faster in most cases.